lib/utils/DevServerPlugin.js

```diff
@@ -252,12 +252,49 @@ class DevServerPlugin {
 
     // use a hook to add entries if available
     if (typeof webpack.EntryPlugin !== 'undefined') {
-      for (const additionalEntry of additionalEntries) {
-        new webpack.EntryPlugin(compiler.context, additionalEntry, {
-          // eslint-disable-next-line no-undefined
-          name: undefined,
-        }).apply(compiler);
-      }
+      // Register Entry Dependency in Factory
+      const EntryDependency = require('webpack/lib/dependencies/EntryDependency');
```

Comment:
Avoid using this, webpack should public export this

Reply:
Should I send PR to webpack then?

Reply:
@alexander-akait 😅